sysutils/pfSense-pkg-LCDproc/files/usr/local/pkg/lcdproc_client.php

```diff
@@ -148,6 +148,48 @@ function lcdproc_get_cpu_temperature() {
 	}
 }
 
+// NTP Status
+function lcdproc_get_ntp_status() { // https://github.com/pfsense/pfsense/blob/master/src/usr/local/www/widgets/widgets/ntp_status.widget.php#L36
+	global $config;
+	if ($config['ntpd']['enable'] == 'enabled') {
+		if (isset($config['system']['ipv6allow'])) {
+			$inet_version = "";
+		} else {
+			$inet_version = " -4";
+		}
+		exec('/usr/local/sbin/ntpq -pnw ' . $inet_version . ' | /usr/bin/tail +3 | /usr/bin/awk -v RS= \'{gsub(/\n[[:space:]][[:space:]]+/," ")}1\'', $ntpq_output);
+		return $ntpq_output;
+	} else {
+		return false;
+	}
+}
+
+function lcdproc_get_ntp_gps_sat_count() {
+	global $config;
+	//if (isset($gps_ok) && isset($config['ntpd']['gps']['extstatus'])) {
+	if (isset($config['ntpd']['gps']['extstatus'])) {
```

Comment:
This would be `if (config_path_enabled('ntpd/gps', 'extstatus')) {`

Reply:
done